src/main/java/org/isf/exa/manager/ExamBrowsingManager.java

```diff
@@ -35,25 +35,20 @@
 import org.springframework.stereotype.Component;
 
 /**
- * Class that provides gui separation from database operations and gives some
- * useful logic manipulations of the dinamic data (memory)
- *
- * @author bob
- * 19-dec-2005
- * 14-jan-2006
+ * Class that provides gui separation from database operations and gives some useful logic manipulations of the dinamic data (memory)
+ * @author bob 19-dec-2005 14-jan-2006
```

Comment:
Author and revision date information is slowly being removed as git history is sufficient